hera_mc/cm_dossier.py

```diff
@@ -187,6 +186,7 @@ def table_row(self, columns, ports):
         list
             A row or rows for the tabulate display.
         """
+        from itertools import zip_longest
```

Comment:
I'm a little confused here. Isn't iterools in the standard libraray? I suppose it doesn't matter too much, but just feels weird to move the import.

Reply:
I agree, I think it should be moved back @david-deboer